docs/guidance/mobilenet-rayservice.md

```diff
@@ -0,0 +1,42 @@
+# RayService: MobileNet example
+
+Both Python scripts for Ray Serve and sending requests are in the repository [kevin85421/ray-serve-examples](https://github.com/kevin85421/ray-serve-examples).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster --image=kindest/node:v1.23.0
+```
+
+## Step 2: Install KubeRay operator
+
+Follow this [document](../../helm-chart/kuberay-operator/README.md) to install the latest stable KubeRay operator via Helm repository.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.mobilenet.yaml
+```
+
+* The [mobilenet.py](https://github.com/kevin85421/ray-serve-examples/blob/main/mobilenet.py) requires `tensorflow` as a dependency. Hence, the YAML file uses `rayproject/ray-ml:2.5.0` instead of `rayproject/ray:2.5.0`.
```

Comment:
```suggestion
* The [mobilenet.py](https://github.com/kevin85421/ray-serve-examples/blob/main/mobilenet.py) file requires `tensorflow` as a dependency. Hence, the YAML file uses `rayproject/ray-ml:2.5.0` instead of `rayproject/ray:2.5.0`.
```

Reply:
Could you also change this link to the `ray-project` repo?

Reply:
I have already removed all personal GitHub link.